index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main class="container">
+        <header class="header-background">
+            <div class="logo-placeholder">
+                <a href="/index.html">Mark</a>
+            </div>
+            <nav class="icon-menu">
+                    <div class="menu-icon">
+                        <img src="./images/menu.svg" alt="menu icon" class="nav-icon">
+                    </div>
+            </nav>
+        </header>
+        <section class="headline">
+            <div class="name">
+                <h1 class="username">I'm Mark, Glad to see you!</h1>
+            </div>
+            <div class="description">
+                <p class="user-description">
+                    I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+            </div>
+            <span class="socialmedia">LET'S CONNECT</span>
```

Comment:
Please use the font family "Poppins" as per requirement for the subtitle. 👍🏻 

[Font Used]
<img width="149" alt="Screen Shot 2022-04-25 at 4 05 49 PM" src="https://user-images.githubusercontent.com/83048208/165077137-9ae17903-27df-4512-ba77-166e826f6a36.png">


[Font Required]
<img width="154" alt="Screen Shot 2022-04-25 at 4 06 13 PM" src="https://user-images.githubusercontent.com/83048208/165077190-bf931a7b-8fa4-415a-a333-a9ec32d273ce.png">